.bazelci/test_buildfarm_container.sh

```diff
@@ -98,6 +101,29 @@ start_server_and_worker(){
     fi
 }
 
+init_grpc_parser(){
+    if [ "${CACHE_TEST:-false}" = true ]; then
+        echo "Fetch tools_remote from git"
+
+        git clone https://github.com/bazelbuild/tools_remote.git
```

Comment:
We can make this reference a specific SHA256 after the tools_remote change lands.

Reply:
Yeah, we can!